### ISSUE TITLE
[BUG] Impossible do the backups with latest version (v2.1.0.2_beta_2024-11-29-ls225)

### ISSUE THREAD

**Issue by PacmanForever** [linuxserver/docker-duplicati#88]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Execute any backup and always appears the same error from the last update.

### Expected Behavior

No errors.

### Steps To Reproduce

- Execute backup
- Wait some minuts
- Appears the error

### Environment

```markdown
Docker in Linux Ubuntu 24.04
```

### CPU architecture

x86-64

### Docker creation

```bash
duplicati:
        restart: unless-stopped
        container_name: duplicati
        ports:
            - '8200:8200'
        environment:
            - PUID=0
            - PGID=0
        volumes:
            - '/dockerData/Duplicati/config:/config'
            - '/dockerData/Duplicati/backups:/backups'
            - '/dockerData:/source'
        networks:
          custom_network:
            ipv4_address: 172.19.0.3
        image: lscr.io/linuxserver/duplicati:latest
```

### Container logs

```bash
One or more errors occurred. (The file duplicati-ieb1bc528c7b4404592c80d5ce3cae324.dindex.zip.aes was uploaded but the returned size was -1 and it was expected to be 7949 (The file duplicati-ieb1bc528c7b4404592c80d5ce3cae324.dindex.zip.aes was uploaded but the returned size was -1 and it was expected to be 7949) (One or more errors occurred. (The file duplicati-ieb1bc528c7b4404592c80d5ce3cae324.dindex.zip.aes was uploaded but the returned size was -1 and it was expected to be 7949)))
```

**Comment by j0nnymoe** [comment]
You need to check duplicati's documentation on how to deal with those errors.

**Comment by PacmanForever** [comment]
Until last weekend everything was fine, the only change was the update to the latest version. To make matters worse, the latest version changed the version of the duplicati database and I can't roll back to the penultimate version. For me this is an issue.

**Comment by j0nnymoe** [comment]
Yes I get this is an issue. We package what duplicati release, this does not look like a container issue to me but an application issue. You need to see if this bug has been reported to duplicati but for my quick googling, it seems a common problem going back years.

**Comment by PacmanForever** [comment]
Okay, I understand that we only need to put container errors here.

**Comment by PacmanForever** [comment]
Just to mention that I put the original image (duplicati/duplicati:latest) and everything works perfectly. Are you sure it's not an error with this image?

**Comment by thespad** [comment]
That image is over 6 months old as they stopped pushing the `latest` tag back in May.

See https://hub.docker.com/r/duplicati/duplicati/tags

**Comment by TobenderZephyr** [comment]
Well, that's not entirely correct
From the README on that repository: 

> Available tags
> [...]
> latest - an alias for beta

Beta has been pushed 9 days ago. This is also when i saw warnings with files being already accessed while doing the backup.

**Comment by thespad** [comment]
They can write whatever they like in the readme, it doesn't change the reality

![image](https://github.com/user-attachments/assets/cbf56b58-15e7-4867-96fd-873e8022a826)

**Comment by kenkendk** [comment]
The missing beta update in Dockerhub is my fault 🤦.
I am pushing to get the first stable release of Duplicati out, and in the process I missed getting the beta == latest tag in there, until the stable release is out.

For @PacmanForever , yes there is a need for a password for the UI in 2.0.9.104+ (so including the beta that is not in Dockerhub:latest). You can set `DUPLICATI__WEBSERVICE_PASSWORD=<password>` to configure it.
More [password details](https://docs.duplicati.com/detailed-descriptions/duplicati-access-password) in docs and a [downgrade to 2.0.8.1 guide](https://docs.duplicati.com/installation-details/release-channels-and-versions/downgrade-from-2.1.0.2-to-2.0.8.1) exists.

**Comment by PacmanForever** [comment]
Since it wasn't working for me, I changed the image, now I use duplicati/duplicati:latest and everything works for me.

Thanks
